controlplane/kubeadm/internal/workload_cluster_coredns_test.go

```diff
@@ -160,25 +204,33 @@ func TestUpdateCoreDNS(t *testing.T) {
 					KubeadmConfigSpec: bootstrapv1.KubeadmConfigSpec{},
 				},
 			},
```

Comment:
Q: could it be that we don't have to change so many of the test cases given that we changed the code again?

Reply:
ack, reverted the unnecessary parts.